src/exact/ExactMapper.cpp

```diff
@@ -5,6 +5,12 @@
 
 #include "exact/ExactMapper.hpp"
 
+#include "Encodings/Encodings.hpp"
+#include "LogicBlock/LogicBlock.hpp"
```

Comment:
This is more of a general remark. From these includes it looks like `Encoding` and `LogicBlock` are part of the QMAP project. However, they originate from the Logicblocks repo. Wouldn't it make sense to place the Logicblocks code in a separate `lb` (or similar) directory similar to how this is done for the DD Package in the QFR for example?